messaging/README.md

```diff
@@ -0,0 +1,6 @@
+## MESSAGING DESIGN PATTERN
+
+Event Driven / Message Stream Based MicroService
+
+
+![img.png](img.png)
```

Comment:
The readme should follow this format : https://github.com/iluwatar/java-design-patterns/wiki/02.-Pattern-template

Reply:
Prefer to generate the class diagram using urm-maven-plugin

Reply:
readme file has been updated based on given template and suggestion